gcb/presubmit.sh

```diff
@@ -59,9 +59,40 @@ bazel_run() {
   fi
 }
 
+check_build_files_for_licenses_rule() {
```

Comment:
Do we need an explicit `return 0` at the end of these functions?

Also, how much additional time do these add to the presubmit runtime?

Reply:
It appears that we do not, but I can add those in a follow up PR just to make sure. I didn't measure the runtime of `presubmit.sh`. My guess is that it's probably faster than the `buildifier` check which actually does parsing, but I do not know for sure.

Reply:
> It appears that we do not, but I can add those in a follow up PR just to make sure. I didn't measure the runtime of `presubmit.sh`. My guess is that it's probably faster than the `buildifier` check which actually does parsing, but I do not know for sure.

OK. I see the presubmit times are between 12 and 25 mins. I think the increase in time is because we keep rebuilding the docker images between the different PRs (due to Dockerfile changes). Once all the Dockerfile changes are in, I hope it comes back to ~7 mins average that we have. Otherwise, we will need to revisit this.